### PR TITLE
Cellscience Updates

### PR DESCRIPTION
**Pull request by imgntn**
This PR changes cellscience to use an AC script to move its entities around, instead of an entity script. 

To test:

1. Visit hifi://cellscience
2. Observe that the cells rotate
3. Click "Inside the Cell" and observe that the vesicles (greyish sphere things) move.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: baa030b7b07a24612953807d5a116398e8354f17 available at: https://gist.github.com/1d4a4767ec946a2e9ec4<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7184.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7184.dmg

**Comment by imgntn** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: baa030b7b07a24612953807d5a116398e8354f17 available at: https://gist.github.com/1d4a4767ec946a2e9ec4<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7184.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7184.dmg

File: unpublishedScripts/DomainContent/CellScience/moveCellsAC.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    }
+
+    results.forEach(function(v) {
+        var name = Entities.getEntityProperties(v, 'name').name;
+      //  print('name is:: ' + name)
+        if (name === 'Cell') {
+         //   print('found a cell!!' + v)
+            Script.setTimeout(function() {
+                moveCell(v);
+            }, Math.random() * THROTTLE_RATE)
```
Comment (ZappoMan) [inline unpublishedScripts/DomainContent/CellScience/moveCellsAC.js]: missing ;

File: unpublishedScripts/DomainContent/CellScience/moveVesiclesAC.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+var sinceLastUpdate = 0;
+
+//print('vesicle script')
+
+function findVesicles() {
+    var results = Entities.findEntities(basePosition, 60000);
+
+    if (results.length === 0) {
+        print('no entities found')
```
Comment (ZappoMan) [inline unpublishedScripts/DomainContent/CellScience/moveVesiclesAC.js]: missing ;

File: unpublishedScripts/DomainContent/CellScience/moveVesiclesAC.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        return;
+    }
+
+    results.forEach(function(v) {
+        var name = Entities.getEntityProperties(v, 'name').name;
+        print('name is:: ' + name)
```
Comment (ZappoMan) [inline unpublishedScripts/DomainContent/CellScience/moveVesiclesAC.js]: missing ;

File: unpublishedScripts/DomainContent/CellScience/moveVesiclesAC.js [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        if (name === 'vesicle') {
+            //    print('found a vesicle!!' + v)
+            Script.setTimeout(function() {
+                moveVesicle(v);
+            }, Math.random() * THROTTLE_RATE)
```
Comment (ZappoMan) [inline unpublishedScripts/DomainContent/CellScience/moveVesiclesAC.js]: missing ;

**Comment by hifi-gustavo** [comment]
Build console output for commit: b0cb4b719994a2e825c7b56f0bf515544e8ef61f available at: https://gist.github.com/1d4a4767ec946a2e9ec4<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7184.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7184.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6c6ea3f0a76ce9bdc433216f3e3bcb6ad5d678e0 available at: https://gist.github.com/1d4a4767ec946a2e9ec4<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7184.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7184.dmg

**Pull request merged**